mgmtfn/k8splugin/README.md

```diff
@@ -0,0 +1,311 @@
+# Contiv Networking for Kubernetes
+
+Contiv is integrated with Kubernetes via a CNI plugin. This integration enables you
+to set up and manage inter-pod connectivity in a Kubernetes cluster using Contiv 
+policy model.
+
+## Getting Started
+
```

Comment:
Reading all the steps looks like something developer friendly instructions when someone wants to hack on this. Should avoid:
- making/building binaries in user instructions
- checking out repos
- ansible work to install things (those all should be part of vbox image, for Vagrant)

We should find hard reasons to not follow what we have working with a swarm cluster i.e. following 4 steps for basic interconnectivity, and a few more for policies, etc.
- 'make demo'
- 'create networks'
- 'create containers'
- 'ping them'

ping @mapuri, @shaleman, @erikh, @unclejack